### PR TITLE
implement Panel list

### PR DESCRIPTION
**Pull request by Ross-Clark**

### CODE REVIEW

File: testapp/home/migrations/0005_review_date.py [hunk 1]
```diff
@@ -28,6 +28,6 @@ class Migration(migrations.Migration):
...
         ),
```
[the remark this answers is not in the document]
Comment (Ross-Clark) [reply testapp/home/migrations/0005_review_date.py]: we were having a problem with source control and migrations, so condensing them into one seemed like a solution to the problem

[the remark this answers is not in the document]
Comment (Ross-Clark) [reply testapp/home/migrations/0005_review_date.py]: we havnt released the project into the wild yet, so no one should be depending on these migrations yet either

[the remark this answers is not in the document]
Comment (Ross-Clark) [reply testapp/home/migrations/0005_review_date.py]: for example I had to merge a change to remove migration 007 when 006 didnt exist in dev today

**Comment by AdamChrimes** [comment]
@Ross-Clark just need to rebase this with `dev` and do anything with the migrations if needed following the hero work being merged in.

**Pull request merged**